### ISSUE TITLE
[Chicken Chunks] Per-player/group chunk limit

### ISSUE THREAD

**Issue by EmanYm** [TheCBProject/Feedback#13]
As requested [here](https://github.com/TheCBProject/ChickenChunks/issues/8)

>In the config is a per-player chunk limit. I am wondering if I need to add each player's name here, or if I can add limits per permission group.

**Comment by covers1624** [comment]
Would need to be per player, I have no way of accessing a bukkit/spogit/sponge plugin and adding permission nodes. Forge now has a permission system, not sure how its used, but ill whack this on the list, Shouldnt really be an issue for servers as they can just limit giving the player a loader if they don't want them loading more chunks.

**Comment by EmanYm** [comment]
I didn't realize this was closed.
My initial idea was kind of a permission node.
something like `chickenchunks.chunklimit.<number>`
then each person in that group would be permitted <number> chunks each.
Then something like `chickenchunks.chunklimit.exempt` on a player (not a group), and another new chunklimit number for that individual.

However.  Since chickenchunks still seems to have an issue on SpongeForge, I've decided to forgo the chunks loader.
